x-pack/plugins/ml/server/models/data_recognizer/modules/security_auth/manifest.json

```diff
@@ -0,0 +1,77 @@
+{
+  "id": "security_auth",
```

Comment:
Looks like the expected response for the auditbeat data set used in the `x-pack/test/api_integration/apis/ml/modules/recognize_module·ts` test needs editing, to add in the ID of the new module `security_auth` which also now matches our test data set. This block here
```
    {
      testTitleSuffix: 'for auditbeat dataset',
      sourceDataArchive: 'x-pack/test/functional/es_archives/ml/module_auditbeat',
      indexPattern: 'ft_module_auditbeat',
      user: USER.ML_POWERUSER,
      expected: {
        responseCode: 200,
        moduleIds: ['auditbeat_process_hosts_ecs', 'security_linux', 'siem_auditbeat'],
      },
    },
```